app/src/ui/diff/code-mirror-host.tsx

```diff
@@ -164,6 +164,12 @@ export class CodeMirrorHost extends React.Component<ICodeMirrorHostProps, {}> {
 
     CodeMirrorHost.updateDoc(this.codeMirror, this.props.value)
     this.resizeObserver.observe(this.codeMirror.getWrapperElement())
+
+    document.addEventListener('dialog-appeared', this.onDialogAppeared)
```

Comment:
From what I see, this event is received by CodeMirror instances that are not inside a dialog, which I guess we don't need/want (it will trigger the `refresh()` of all rendered CodeMirror instances when _any_ dialog appears).

Not sure if there is an easy/performant way of narrowing down this event to only child elements of a dialog, other than replacing that `document` with the closest `dialog` ancestor.